pitch.md

```diff
@@ -7,14 +7,28 @@ _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+My language is for people who want to draw tableau trees, more specifically for students in CS 81.
+CS 81 requires trees that are rendered digitally and the recommended method for drawing the trees is to use Google Sheets.
+So students have to create text-boxes and arrows and drag them around to draw tree, which is quite cumbersome.
+A common alternative is to use LaTeX to typeset a tree.
+However, not everyone knows how to use LaTeX because the syntax is a bit challenging (and requires a lot of boilerplate). 
+Even if someone uses LaTeX for their other classes, they may not be familiar with the LaTeX packages for typesetting trees.
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+Logicians already use a DSL, LaTeX, for this purpose.
+A language has a steeper learning curve compared to using Google Sheets, but once someone has learned a language, it should be much easier to use the language than drawing a diagram.
+It seems like another DSL, with a simpler interface compared to LaTeX can improve a user's experience.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm taking CS 81 and I really dislike drawing trees on Google Sheets.
```

Comment:
"Was there a need that was similar to one you thought of? If so, how are your thoughts about the problems similar to and different from your partners?" This is not really a need that is similar to one that I thought of -- it's kind of similar to Farhan's idea in that it lets a user create a tree/graph using a language instead of dragging around different text boxes and lines. For my idea of parsing chord sheets to a language that LilyPond can read, I don't have to worry about how exactly LilyPond creates the images of the sheet music and chords. My DSL is mostly concerned with parsing the name of a chord -- given in many possible different formats, depending on the level of music theory that a user understands / the amount of time a user wants to spend in specifying very specific, inverted chords -- into the input that LilyPond requires so that it can generate sheet music. Renata's language is more concerned with actually having to figure out how the language maps onto the graphical presentation / orientation of different elements of a tableau tree.